src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const checkStatus = () => ({
+  type: 'CHECK_STATUS',
+  payload: {
+    status: 'online',
+  },
+});
+
+const categoriesReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return 'Under construction';
+    default:
+      return state;
+  }
+};
+
+export { checkStatus };
+export default categoriesReducer;
```

Comment:
- [ ] in this file you should Define CHECK_STATUS as action type, please keep in mind that The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)